internal/controller/spinapp_controller.go

```diff
@@ -423,7 +423,7 @@ func constructDeployment(ctx context.Context, app *spinv1alpha1.SpinApp, config
 		Requests: app.Spec.Resources.Requests,
 	}
 
-	env := ConstructEnvForApp(ctx, app)
+	env := ConstructEnvForApp(ctx, app, spinapp.DefaultHTTPPort)
```

Comment:
so in future we may add the ability to override the port using SpinApp CRD?

Reply:
I think we'll want to keep it "automatic", but it'll at least be able to be non-80 😅